script_test.go

```diff
@@ -1219,6 +1220,22 @@ func TestExecRunsGoHelpAndGetsUsageMessage(t *testing.T) {
 	}
 }
 
+func TestExecContextCencel(t *testing.T) {
+	t.Parallel()
+	// We can't make many cross-platform assumptions about what external
+	// commands will be available, but it seems logical that 'go' would be
+	// (though it may not be in the user's path)
+	ctx, cancel := context.WithTimeout(context.Background(), 1)
+	defer cancel()
+	p := script.ExecContext(ctx, "sleep 100")
```

Comment:
```suggestion
	p := script.ExecContext(ctx, "sleep 3")
```

This is more than enough